docs/rc.xml.all

```diff
@@ -30,6 +30,7 @@
   <theme>
     <name></name>
     <icon></icon>
+    <fallbackAppIcon>labwc</fallbackAppIcon>
```

Comment:
We could consider using `image-missing` as default but it fails to load for me even though its available in multiple themes. Not quite sure why, maybe we need to tell libsfdo about the used theme?